.travis.yml

```diff
@@ -31,3 +36,4 @@ cache:
 branches:
   only:
   - master
+  - use-new-uac-qid-service
```

Comment:
Needs removing, shouldn't need to specify branches here since it's building off PR's.